how-to/how-to-load-kernel-modules-with-kata.md

```diff
@@ -0,0 +1,98 @@
+# Loading kernel modules
+
+A new feature for loading kernel modules was introduced in Kata Containers 1.9.
+The list of kernel modules and their parameters can be provided using the
+configuration file or OCI annotations. The [Kata runtime][1] gives that
+information to the [Kata Agent][2] through gRPC when the sandbox is created.
+The [Kata Agent][2] will insert the kernel modules using `modprobe`, hence
+modules dependencies are resolved automatically.
+
+The sandbox will not be started when:
+
+  * A kernel module is specified and the `modprobe` command is not installed in
+    the guest or it fails loading the module.
+  * The module is not available in the guest or it doesn't met the guest kernel
+    requirements, like architecture and version.
+
+In the following sections are documented the different ways that exist for
+loading kernel modules in Kata Containers.
+
+- [Using Kata Configuration file](#using-kata-configuration-file)
+- [Using annotations](#using-annotations)
+
+# Using Kata Configuration file
+
+The list of kernel modules and parameters can be set in the `kernel_modules`
```

Comment:
Can you add a note here: "Use this method, only if you need to pass the kernel modules to all containers. Please use annotations described below to set per pod annotations"
Or best, I would put this section for the configuration file below annotations itself, since we would expect users to mostly use annotations.